### PR TITLE
SUMO-216336 - Update RBAC for indexes beta article 

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) updates the following beta article per reviewer comments:
https://help.sumologic.com/docs/manage/users-roles/roles/rbac-for-indexes

Issue number: [SUMO-216336](https://sumologic.atlassian.net/browse/SUMO-216336)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/manage/users-roles/roles/rbac-for-indexes.md [hunk 1]
```diff
@@ -63,6 +66,14 @@ An advanced filter allows access only to the logs that match the search filter.
    * **Security data filter**. This filter applies on all logs in [Cloud SIEM security indexes](/docs/cse/records-signals-entities-insights/search-cse-records-in-sumo#partitions-for-cse-records).
 1. Enter search criteria in the box provided. For examples, see [Understanding search filters](/docs/manage/users-roles/roles/construct-search-filter-for-role#understanding-search-filters).<br/><img src={useBaseUrl('img/users-roles/advanced-filter.png')} alt="Advanced filter" style={{border: '1px solid black'}} width="400"/>
 
+### Advanced filter examples
+
+Following are examples for advanced filtering:
+* Let’s say you want to deny access to all logs that contain “error” in log analytics, and contain “malicious=high” in security logs. Select **Log Analytics data filter** and add `!error` to the filter, and then select **Security data filter** and add `!malicious=high` to the filter.
```
Comment (kimsauce) [inline docs/manage/users-roles/roles/rbac-for-indexes.md:72]: should these be in monospace, too?

```suggestion
* Let’s say you want to deny access to all logs that contain `error` in log analytics, and contain `malicious=high` in security logs. Select **Log Analytics data filter** and add `!error` to the filter, and then select **Security data filter** and add `!malicious=high` to the filter. 
```

Comment (jpipkin1) [reply docs/manage/users-roles/roles/rbac-for-indexes.md]: Yes, @kimsauce. That's a good suggestion.

**Review by kimsauce — APPROVED** [review]

File: docs/manage/users-roles/roles/rbac-for-indexes.md [hunk 2]
```diff
@@ -50,6 +50,9 @@ An index based search filter allows or denies access to [search indexes](/docs/m
    * **Deny few indexes**. Deny access to the selected indexes. 
 1. If you choose **Allow few indexes** or **Deny few indexes**, choose the indexes in the **Select Indexes** box that appear.<br/><img src={useBaseUrl('img/users-roles/index-based-filter.png')} alt="Index based filter" style={{border: '1px solid black'}} width="400"/>
    
+### Index based filter example
+
+For example, let’s say you want to deny access to partition and security indexes. In our example environment, the “accessLogs” and “authenticationLogs” indexes give access to partitions, and the “sec_*” indexes give access to security information. To deny access to these indexes, click **Deny few indexes** and select those indexes.
```
Comment (kimsauce) [inline docs/manage/users-roles/roles/rbac-for-indexes.md:55]:
```suggestion
For example, let’s say you want to deny access to partition and security indexes. In our example environment, the `accessLogs` and `authenticationLogs` indexes give access to partitions, and the “sec_*” indexes give access to security information. To deny access to these indexes, click **Deny few indexes** and select those indexes. 
```

Comment (kimsauce) [reply docs/manage/users-roles/roles/rbac-for-indexes.md:55]: Sorry @jpipkin1 , I found one more possible suggestion!

Comment (jpipkin1) [reply docs/manage/users-roles/roles/rbac-for-indexes.md:55]: Good suggestion, @kimsauce.

**Pull request merged**